utils/fetch.ts

```
@@ -6,10 +6,20 @@ interface Request<T> {
+}
+
 class Https {
-  BASE_URL: string = import.meta.env.VITE_BASE_URL || ''
+  BASE_URL: string = import.meta.env.VITE_BASE_API || ''
   authToken: string = ''
-  constructor() {}
+  constructor() {
+    // if (import.meta.dev) {
+    //   this.BASE_URL = '/proxy'
+    // }
+  }
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)